internal/cloud/realm/access_list.go

```diff
@@ -42,7 +71,7 @@ func (c *client) AllowedIPCreate(groupID, appID, ipAddress, comment string, useC
 	}
 
 	if res.StatusCode != http.StatusCreated {
-		return AllowedIP{}, api.ErrUnexpectedStatusCode{"create allowed ip", res.StatusCode}
+		return AllowedIP{}, api.ErrUnexpectedStatusCode{"create allowed ip or CIDR block", res.StatusCode}
```

Comment:
sorry it seems like i'm going back and forth on this, but these errors are not intended to be user facing, we can keep them concise and relevant to the specific context of the client call itself here

```suggestion
		return AllowedIP{}, api.ErrUnexpectedStatusCode{"create allowed ip", res.StatusCode}
```